sdks/go/test/integration/integration.go

```diff
@@ -67,6 +67,8 @@ var directFilters = []string{
 	"TestTestStream.*",
 	// (BEAM-13075): The direct runner does not support windowed side inputs
 	"TestValidateWindowedSideInputs",
+	// The direct runner does not currently support multimap side inputs
```

Comment:
Please add the JIRA for this issue. Likely the work can be consolidated w/ the windowed side inputs above, as it'll require similar/identical work.

Scope creep, but very related scope creep.

Reply:
Done. I also updated the JIRA to cover both side input changes in the direct runner